.github/workflows/main.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     strategy:
       matrix:
         os: [ macos-latest, windows-latest, ubuntu-latest ]
-        arch: [ x64, arm64 ]
```

Comment:
no more arm64 ?

Reply:
It's a WIP sorry I forgot to mention it.
It's not uploading in the right directory for now 😅